docs/changelog.md

```diff
@@ -37,6 +37,68 @@ It doesn't include
 ## Unreleased
 
 
+### Highlights for users
+
+* Initial support for muted users. (PRs #4774, #4730, #4710, #4736, #4711,
+  #4699, #4678)
+* Added a setting to choose whether external links should open in the
+  embedded browser or the external browser. (PR #4679)
+* Added a server-compatibility notice for self-hosted orgs running Zulip
+  Server versions before 2.0.0. (PR #4750)
+* Stream settings screen now has a "subscribe" button. And, when
+  unsubscribed, useless toggle switches ("Pinned", "Muted", and
+  "Notifications") are hidden. (#4715)
+* Updated the primary app color to match the new logo. (PR #4544)
+* Fixed a bug on Android where the app sometimes opened to an unexpected
+  narrow when opening the app from the background. (#4758)
+* Fixed a glitch where the org URL input screen would flash by before the
+  authentication screen, on some app startups or when the user tapped a
+  logged-out account in the account picker. (#4604)
+* Fixed misconfigurations causing the Play Store to not offer the Android
+  app on devices that don't have cameras (#4722) or WiFi capability (PR
+  #4761).
+* Continued work toward enabling Android share-to-Zulip. (#4514)
+* Unclear, little-used "create group PM" option in the compose box removed.
+  (PR #4777)
```

Comment:
This doesn't go in this section -- most users probably never noticed that icon, and nobody will be excited it's gone. (Some will be pleased it's gone, from a UI-connoisseur perspective; but from the direct user perspective, it's the removal of an annoyance one hadn't even quite put a finger on.)